src/views/download/download.scss

```diff
@@ -0,0 +1,84 @@
+@import "../../colors";
+@import "../../frameless";
+
+$developer-spot: $ui-yellow;
+
+#view {
+    padding: 0;
+}
+
+.download {
+    .title-banner {
+        &.masthead {
+            background-color: $developer-spot;
+            padding-bottom: 0;
+
+            h1 {
+                margin: 0 0 2rem 0;
+                text-align: left;
+                color: $ui-white;
+            }
+
+            p {
+                margin: 0;
+                width: $cols6;
```

Comment:
This will cause some overflow at the mobile level, because this width is larger than the screen width – it would be good to add an override to this in the mobile breakpoint